arista/cvp/plugins/modules/cv_configlet.py

```diff
@@ -179,6 +179,7 @@ def configlet_action(module):
     new_configlet = []  # configlets to add to CVP
     new = []
     taskList = []  # Tasks that have a pending status after function runs
```

Comment:
do we still need this?

Reply:
It will be reused to centralized all `taksId` collecting during module execution.